### ISSUE TITLE
Is Nuclear energy climate friendly ?

### ISSUE THREAD

**Issue by Calmwaters77** [g4challenge/ds4ns#12]
**Author**

- **Name**: Oliver Zeilerbauer

- **Affiliation**: FH Kufstein

**Keywords**
Nuclear Energy, Nuclear Fission, Small Modular Reactor’s (SMR’s), IAEA (International Atomic Energy Agency), IEA (International Energy Agency)

**Abstract**
A noticeable amount of the global total energy is supplied by nuclear energy. A Chart from the IEA (International Energy Agency) shows that it increased from 0.9% in 1973 to 5.0% in 2019.[1] Although not a renewable energy source, nuclear energy counts as a low-carbon energy, since during the process the CO2-emissions are near zero. For a comparison of the CO2-emissions the lifecycle of the energy source is tracked, namely manufacturing, transportation and production processes. While Renewable Energies like wind and solar are heavily influenced by weather conditions which limits the places where you can build them, nuclear energy has less restrictions and can be built in a lot more spaces, safety measures uphold. Since some communities cannot use or store as much energy as a big powerplant produces, there are smaller and thus cheaper architectures in research so called SMR’s (Small Modular Reactors). Small is self-explanatory, they are smaller than big power plants. Modular in the sense that you can stack them together to make a big power plant.

[Nuclear_Energy_oliver_zeilerbauer.pdf](https://github.com/g4challenge/ds4ns/files/10520977/NaturalScience_oliver_zeilerbauer.pdf)

Full PDF: DOI, [Zenodo], 
[Github Repo](https://github.com/g4challenge/ds4ns_template).
[Github Release](https://github.com/g4challenge/ds4ns_template/releases/tag/test_release)
[Zenodo WholeTale](https://sandbox.zenodo.org/record/964368)
[Zenodo Github](https://sandbox.zenodo.org/record/964372)
.. [![DOI](https://zenodo.org/badge/DOI/10.5281/zenodo.4661161.svg)](https://doi.org/10.5281/zenodo.4661161)

**Comment by kayagoekan** [comment]
The author Oliver Zeilerbauer has dealt intensively with the topic of nuclear energy in relation to its future application. In doing so, the author addressed the questions of whether nuclear energy is climate-friendly or not. 
Global, political decisions and agreements were taken into consideration, among other things. Furthermore, the damage and the disposal of nuclear waste are discussed in detail. 
In conclusion, one can read very well from the article that in the future very good alternatives like Small Modular Reactors (SMRs) would be a good alternative to produce climate-friendly energy. In time, it might be possible to patent these small modular reactors. One of the big advantages according to studies in the article are among others high cost reductions and offers high safety compared to classical reactors. 
This article is a valuable contribution for future decisions and discussions on nuclear energy climate-friendly useful or not. The article is very well structured and the language is very easy to understand.
Points (20/20)

**Comment by TinyTen** [comment]
The author Oliver Zeilerbauer has summarized sustainability and safety aspects of nuclear power plants in comparison with non-renewable energy sources.
The author presents an interesting view on the topic and concludes that nuclear energy is at least way more sustainable than fossil fuels. He supports his conclusion with several numbers and analyses the topic from different perspectives. Regarding the novelty, the paper reports about new technologies and findings. No own research is conducted which is acceptable for such a short student work. The topic is extremely relevant and will remain a topic of discussion for a very long time.
Since the paper is created for a data science conference, I expected to find information on the connection of the topic with data science. The lack of such a connection is a shortcoming of the paper.
Regarding the reference, the author presents links which mainly include scientific papers and websites or reports from official agencies. These sources might be scientifically valuable but need to be cited according to the standards. An example would be:


Krall, L. M., Macfarlane, A. M., & Ewing, R. C. (2022). Nuclear waste from small modular reactors. Proceedings of the National Academy of Sciences, 119(23), e2111833119.


Direct quotes should also contain the page number where the sentence can be found.
The style of the paper does not yet totally match the ACM style guide (https://www.acm.org/publications/proceedings-template). Images should be moved in order to close gaps in the text and should carry a separate number and caption that can be referenced in the text. The abstract is more a introduction than a summary hence the title of the section should be changed.
Regarding spelling and writing style I would advise to be aware of uppercase and lowercase letters and verbforms (in regard to singular/plural forms). In scientific writing I would also prefer a more passive style such as ‘As can be seen’ instead of ‘As we can see’. 
In conclusion, the paper presents a very interesting topic and an overview of recent literature. In terms of style and scientific representation several points need to be revised as described above.
Points (12/20)
